ci_framework/playbooks/01-bootstrap.yml

```diff
@@ -19,7 +19,7 @@
           {{
             hostvars[inventory_hostname] |
             dict2items |
-            selectattr("key", "match", "^(cifmw|pre|post)_(?!install_yamls|openshift).*") |
+            selectattr("key", "match", "^(cifmw|pre|post)_(?!install_yamls|openshift_login).*") |
```

Comment:
We may want to NOT save the token?
`cifmw_openshift_token: sha256~hZeQTUxbZujON4uUcaNmryNz7eMxsF6txXnL6aY3w0g
`

Reply:
correct